oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -426,6 +428,20 @@ private void checkIfStopped() throws CommitFailedException {
 
     @Override
     public synchronized void run() {
+        boolean isIndexWithLanePresent = false;
+        NodeState oakIndexNode = store.getRoot().getChildNode("oak:index");
+        Iterator<? extends ChildNodeEntry> indexNodeIterator = oakIndexNode.getChildNodeEntries().iterator();
+        while (indexNodeIterator.hasNext()){
+            ChildNodeEntry  childNodeEntry = indexNodeIterator.next();
+            if (childNodeEntry.getNodeState().getProperty("async") != null){
+                isIndexWithLanePresent = true;
+                break;
+            }
+        }
+        if(!isIndexWithLanePresent){
+            log.info("lane: {} not present for indexes under /oak:index", name);
```

Comment:
"name" is the name of the lane, right? that could be "elastic-async" right?

Reply:
yes